flink-tests/src/test/java/org/apache/flink/test/checkpointing/ResumeCheckpointManuallyITCase.java

```diff
@@ -94,71 +94,62 @@ public static Object[] parameters() {
     @Test
     public void testExternalizedIncrementalRocksDBCheckpointsStandalone() throws Exception {
         final File checkpointDir = temporaryFolder.newFolder();
+        StateBackend stateBackend = createRocksDBStateBackend(checkpointDir, true);
         testExternalizedCheckpoints(
-                checkpointDir,
-                null,
-                createRocksDBStateBackend(checkpointDir, true),
-                false,
-                restoreMode);
+                checkpointDir, null, stateBackend, false, restoreMode);
```

Comment:
I don't think we need to change all existing tests since we introduced the wrapper `testExternalizedCheckpoints` method.